docs/output.md

```diff
@@ -101,23 +101,57 @@ The files will be phased if the flag `--phase` is used (with `_phased` suffix).
 
 ## Imputation outputs `--steps impute`
 
-The results from steps impute will have the following directory structure:
+The results from `--steps impute` will have the following directory structure:
```

Comment:
We could add the tree directory structure like we did with --panelprep

Reply:
Done